src/control/cam_control_mod.meta

```diff
@@ -0,0 +1,32 @@
+[ccpp-table-properties]
+  name = cam_control_mod
+  type = module
+
+[ccpp-arg-table]
+  name  = cam_control_mod
+  type  = module
+[ aqua_planet ]
+  standard_name = do_aqua_planet
+  units = 1
+  type = logical
+  dimensions = ()
+[ eccen ]
+  standard_name = eccentricity_factor
+  units = 1
+  type = real | kind = r8
+  dimensions = ()
+[ obliqr ]
+  standard_name = obliquity
+  units = radians
```

Comment:
Is radians spelled out in the CCPP?  According to SI the unit label is `rad`.

Reply:
I also prefer `rad` since the other units are abbreviated. The dictionary has both "rad" and "radians" as units. In atmospheric_physics, we use "radians", so I'll make an issue to change those

Reply:
do we also want to change:
`degrees` -> `deg`
`steradian` -> `sr`

Reply:
per our discussions last week, i have update radians to rad and steradian to sr